integration/kubernetes/helm-chart/monitor/templates/prometheus/deployment.yaml

```diff
@@ -0,0 +1,84 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{ if .Values.prometheus.enabled -}}
+{{- $hostNetwork := .Values.prometheus.hostNetwork }}
+{{- $fullName := include "monitor.fullname" . }}
+{{- $hostPID := .Values.prometheus.hostPID }}
+{{- $name := include "monitor.name" . }}
+{{- $chart := include "monitor.chart" . }}
+
+apiVersion: apps/v1
```

Comment:
1 or v1 or `apps/v1`?

Reply:
Here is deployment.yaml, "apps/v1" is fine and remain same with alluxio chart